src/OutputFormatter/ConsoleOutputFormatter.php

```diff
@@ -30,7 +38,7 @@ public function configureOptions(): array
 
     public function enabledByDefault(): bool
     {
-        return true;
+        return !(false !== $this->env->get('GITHUB_ACTIONS'));
```

Comment:
```suggestion
        return false === $this->env->get('GITHUB_ACTIONS');
```

This should give the same result without negations and because of this more readable